### DISCUSSION TITLE
[Feature] Built-in backup and restore

### DISCUSSION THREAD

**Discussion by temal32** [immich-app/immich#12362] [Feature Request]
### I have searched the existing feature requests to make sure this is not a duplicate request.

- [X] Yes

### The feature

Hello,
first of all, **Immich is great**! I've been using it for around 1 month now and it's the best selfhosting photo and video solution in my opinion. 

I was thinking of a feature which is included in the Immich Docker installation, a feature to backup and possibly to restore. The backup automatically gets the entire library folder + .env file + sql dump file + docker-compose.yml file (for rebuilding on a new machine). 

I suggest this feature because it's not "easy" to create a backup and restore from it currently.

### Platform

- [X] Server
- [ ] Web
- [ ] Mobile

**Comment by mmomjian** [comment]
Thank you. Duplicate of #5441

**Comment by temal32** [comment]
@mmomjian I'm not talking about creating and restoring a backup via GUI, i mean by CLI.

**Reply by mmomjian** [reply]
Ok. You can comment that on the linked request if you want. The ideas are very similar in practice so they can be tracked together.
